### DISCUSSION TITLE
Is there a list of screens supported by this fork?

### DISCUSSION THREAD

**Discussion by spitfire** [jayofelony/pwnagotchi#18] [Q&A]
Release notes for 2.7.3 state that "This fork now supports up-to 70 Waveshare e-ink displays.", but where can I find a list of them? I'm looking for something bigger than zero-sized ones, but would still fit the Rpi3A form factor.

**Comment by jayofelony** [comment]
https://github.com/waveshareteam/e-Paper/tree/master/RaspberryPi_JetsonNano/python/lib/waveshare_epd
It supports all of these. Best for a 3 or 4 would be 2.7 inch or 2.9 inch I think. Maybe a 3.5 inch LCD.

**Reply by spitfire** [reply]
Thanks for the info! I'm looking for the screen fitting [Raspberry Pi 3A](https://www.raspberrypi.com/products/raspberry-pi-3-model-a-plus/)  (its form factor is closer to a square than a rectangle, not the (larger) B model
